mino/proxy/http/controller/mod.go

```diff
@@ -32,6 +34,17 @@ func (m minimal) SetCommands(builder node.Builder) {
 		Value:    defaultAddr,
 	})
 	sub.SetAction(builder.MakeAction(startAction{}))
+
+	sub = cmd.SetSubCommand("prom")
```

Comment:
my understanding is that here we're only configuring the metrics path, rather than the port on which we're exposing the metrics.
We may not want to expose metrics on the same port as the application-level data.
If they're on the same port, firewalls can't (trivially) let different sources access one but not the other.
What do you think?

Reply:
I put this command as a convenience to have a server quickly up and running. For a production case, the one where firewall rules matter for example, then it is expected that the user implement its own server.

Reply:
Would that then be the case in d-voting, for instance?

Reply:
Yes, in the case of d-voting we will add a small http server in our implementation to handle the Prometheus metrics. The idea is to keep Dela simple, with minimal functionalities.